cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,32 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('/#/login?_k=r8hegh');
+  });
   it('should provide an ability to log in', () => {
+    const email = 'gogoduck@qa.team';
```

Comment:
Hardcoded login credentials. Consider using environment variables or a separate config file for such kind of sensitive data

Reply:
Understood